quirks/password-rules.json

```diff
@@ -20,6 +20,9 @@
     "angieslist.com": {
         "password-rules": "minlength: 6; maxlength: 15;"
     },
+    "argosco.uk": {
+        "password-rules": "minlength: 6; required: lower, upper, digit;"
```

Comment:
Is there a typo in this URL? [argosco.uk](argosco.uk) gives me an error message (404). Shouldn't this be [argos.co.uk](argos.co.uk) as stated in the title of pull request?

Reply:
Just pushed new commit.